src/deprecation.js

```diff
@@ -0,0 +1,37 @@
+var core = require('./core'),
+    math = core.math,
+    // plugins:
+    extras         = require('./extras'),
+    filters        = require('./filters'),
+    interaction    = require('./interaction'),
+    loaders        = require('./loaders'),
+    spine          = require('./spine'),
+    text           = require('./text');
+
```

Comment:
I bet jshint is giving you unused variable warnings in this file, I think only 2 of these requires are being used. Make sure to run jshint before committing.